pkg/chains/signing.go

```diff
@@ -248,3 +250,23 @@ func HandleRetry(ctx context.Context, obj objects.TektonObject, ps versioned.Int
 	}
 	return MarkFailed(ctx, obj, ps, annotations)
 }
+
+// getRawPayload returns the payload as a json string. If the given payload is a proto.Message type, protojson.Marshal
+// is used to get the proper labels/field names in the resulting json.
+func getRawPayload(payload interface{}) ([]byte, error) {
```

Comment:
Can we test this function by any chance?

Reply:
I saw some other places where private functions are unit-tested, so I follow the same pattern and added a unit-test for this function, let me know if that's what you had in mind. Thanks!

Reply:
LGTM!